pkgs/top-level/python-packages.nix

```diff
@@ -11319,6 +11319,38 @@ in {
     };
   });
 
+  notify2 = buildPythonPackage rec  {
```

Comment:
Please add this to a dedicated file in `python-modules` and add a reference here with callPackages as described in the header of this file.